packages/push-cordova/README.md

```diff
@@ -0,0 +1,19 @@
+## AeroGear Metrics Cordova plugin
+
+Wrapper for existing push plugin phonegap-plugin-push
+
+See push plugin documentation for more details:
+https://github.com/phonegap/phonegap-plugin-push
+
+
+## Installation
+
+```
+cordova plugin add @aerogear/cordova-plugin-aerogear-push
```

Comment:
Any reason why not use the conversation aerogear-[plataform]-[feature]?

eg: aerogear-cordova-push-plugin

Reply:
If you mean `convention` we use cordova plugin convention for discoverability in npm.